packages/thumbnail/src/Thumbnail.ts

```diff
@@ -13,23 +13,38 @@ governing permissions and limitations under the License.
 import {
     CSSResultArray,
     html,
-    SizedMixin,
     SpectrumElement,
     TemplateResult,
 } from '@spectrum-web-components/base';
 import { property } from '@spectrum-web-components/base/src/decorators.js';
 
 import styles from './thumbnail.css.js';
 
+export type ThumbnailSize =
```

Comment:
Would be good to know if design is expecting more patterns to leverage numbered t-shirt sizing. If so, we could add this as a specific feature of the `sizedMixin` to ensure that we're not doing so much copy & pasting over time.

Reply:
I did touch base with design, it sounds like for now it is just Avatar and Thumbnail using this scale.